test/spec-example/shmem_team_split_2D.c

```diff
@@ -30,7 +29,10 @@ static void find_xyz_dims(int npes, int *x, int *y, int *z) {
 }
 
 int main(void) {
-  int xdim, ydim, zdim;
+  //int xdim, ydim, zdim;
+  int xdim = 0;
+  int ydim = 0;
+  int zdim = 0;
```

Comment:
I think setting these to `1` might be a safer choice.  Can you please make this one last change @rdesai16?